spec/features/posts_spec.rb

```diff
@@ -6,6 +6,7 @@
 
     fill_in 'post_title', with: 'Title'
     fill_in 'post_body',  with: 'Body'
+    select 'draft', from: 'post_status'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.